problems/fibonacci.yaml

```diff
@@ -0,0 +1,31 @@
+id: 2
+title: Fibonacci
+text: |
+  Write a function called Fibonacci that takes in an integer.
+  given an input N, this function should return the Nth number in the Fibonacci sequence.
+  
+  Fibonacci sequence is defined as
+  F(0) = 0   
+  F(1) = 1
+  F(N) = F(N - 1) + F(N - 2), for N > 1.
+
+
+Example:
```

Comment:
did you mean to add this line? it adds a property called example w/ a value of null to datastore

Reply:
Whoops, thanks for the catch!
I think it got pasted by accident, deleting it.